modelhub/modelhub/pipelines/extracted_contexts.py

```diff
@@ -99,20 +101,81 @@ class ExtractedContextsPipeline(BaseDataPipeline):
     def __init__(self, engine: Engine, table_name: str, global_contexts: List[str]):
         super().__init__()
         self._engine = engine
-        self._table_name = table_name
-        self._taxonomy_column = _get_taxonomy_column_definition(engine)
+        self._db_dialect = DBDialect.from_engine(self._engine)
         self._global_contexts = global_contexts
+        self._table_name = table_name
+        self._taxonomy_column: Optional[TaxonomyColumnDefinition] = None
 
         # check if table has all required columns for pipeline
         dtypes = bach.from_database.get_dtypes_from_table(
             engine=self._engine,
             table_name=self._table_name,
         )
+
+        # If the taxonomy column is present, we need to use the "old" format
+        if _get_taxonomy_column_definition(engine).name in dtypes:
+            self._taxonomy_column = _get_taxonomy_column_definition(engine)
```

Comment:
taxonomy_column = _get_taxonomy_column_definition(engine)
```suggestion
        if taxonomy_column.name in dtypes:
            self._taxonomy_column = taxonomy_column
```